### PR TITLE
fix(components): [virtual-list]fix namespace in virtual-list

### LINKED ISSUE #8254: [Bug Report] Namespace doesn't work on class el-vl__window
### Element Plus version
2.2.5

### OS/Browsers version
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36

### Vue version
3.2.37

### Reproduction Link
[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgcmVhY3RpdmUsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgRXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcblxuY29uc3QgbXNnID0gcmVmKCcnKVxuY29uc3Qgb3B0aW9ucyAgPSBBcnJheSgxMDAwKS5maWxsKDApLm1hcCgoXywgaW5kZXgpID0+ICh7IGxhYmVsOiBTdHJpbmcoaW5kZXgpLCB2YWx1ZTogaW5kZXggfSkpXG5jb25zdCBjb3VudCA9IHJlZigxMClcbmZ1bmN0aW9uIGxvYWQoKSB7XG4gIGNvdW50LnZhbHVlICs9IDEwXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtY29uZmlnLXByb3ZpZGVyIG5hbWVzcGFjZT1cImVwXCIgPlxuXHRcdFx0PGVsLXNlbGVjdC12MiB2LW1vZGVsPVwibXNnXCIgOm9wdGlvbnM9XCJvcHRpb25zXCIgLz5cbiAgIDwvZWwtY29uZmlnLXByb3ZpZGVyPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==)
### Steps to reproduce
1. change namespace to 'eb' and use ElSelectV2
2. click select to dropdown
3. find the class of el-vl__window didn't change

### What is Expected?
the class el-vl__window changes with namespace

### What is actually happening?
the class el-vl__window doesn't change with namespace

### PR DESCRIPTION
**Pull request by FlareZh**
fix #8254 ,changed el-vl__window to support namespace

### CODE REVIEW

File: packages/components/virtual-list/src/builders/build-list.ts [hunk 1]
```diff
@@ -514,7 +514,7 @@ const createList = ({
       const listContainer = h(
         Container as VNode,
         {
-          class: ['el-vl__window', className],
+          class: [`${ns.namespace.value}-vl__window`, className],
```
Comment (tolking) [inline packages/components/virtual-list/src/builders/build-list.ts:517]:
```diff
- class: [`${ns.namespace.value}-vl__window`, className],
+ class: [ns.e('window'), className],
```

**Comment by tolking** [comment]
@FlareStone seems like your commit message contains some issues. You should modify the git commits history message

**Comment by FlareStone** [comment]
> @FlareStone seems like your commit message contains some issues. You should modify the git commits history message

Thank you, is it correct now? How could I reactivate the github-actions bot to check my commit message?

**Review by tolking — APPROVED** [review]

**Comment by tolking** [comment]
Thanks for your contribution

**Pull request merged**